admin_guide/managing_projects.adoc

```diff
@@ -376,3 +376,48 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-limits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+
+.Example Project Limit for Users
+
+----
+openshift.io/project-limit: '0'
+openshift.io/project-limit: '1'
+openshift.io/project-limit: '3'
+openshift.io/project-limit: '5'
+openshift.io/project-limit: '10'
+openshift.io/project-limit: '20'
+openshift.io/project-limit: '50'
+openshift.io/project-limit: '100'
+openshift.io/project-limit: 'unlimited'
```

Comment:
These are not just the example values, but the ONLY values acceptable. We need to make that clear.